### PR TITLE
By referral

### PR DESCRIPTION
**Pull request by Beiancaa**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,7 +44,7 @@ gem 'datatables-rails', '~> 1.10.7.0'
 gem "select2-rails"
 gem 'jquery-turbolinks'
 gem 'pointless_feedback'
-gem 'dependent-fields-rails'
+gem 'clipboard-rails'
```
Comment (nooralnabi) [inline Gemfile]: Where do you use this gem?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -25,16 +25,19 @@
...
     $('.js-example-basic-multiple').select2();
 });
 
-$(document).on("turbolinks:load", function () {
-  DependentFields.bind()
-});
+$(document).ready(function(){  
+  
+  var clipboard = new Clipboard('.clipboard-btn');
+  console.log(clipboard);
```
Comment (nooralnabi) [inline app/assets/javascripts/application.js]: do you need to print the value into the console

File: Gemfile (hunk 1 above)
Comment (Beiancaa) [reply Gemfile]: In the new modal..

Comment (MwMaki) [reply Gemfile]: @Beiancaa Is this gem only for copy and past the referral code?

**Comment by nooralnabi** [comment]
@Beiancaa, please check  the range slider on creating a new ad in a business page is not working

**Pull request merged**